CONTRIBUTING.md

```diff
@@ -29,6 +62,6 @@ tox creates virtual environment for every tox environment defined in
 source .tox/unit/bin/activate
 ```
 
-## Integration tests
+**Integration tests**
```

Comment:
I don't think we need these 2 lines until they are actually implemented

Reply:
Removed.